core/utility/constants.py

```diff
@@ -23,7 +23,14 @@
 CONSTANTS_JS_FILEPATH = os.path.join('static', 'assets', 'constants.js')
 
 def parse_json_from_js(js_file):
-    """Extracts JSON object from JS file."""
+    """Extracts JSON object from JS file.
+    Args:
+        js_file: file. A stream handling object of constants.js.
```

Comment:
"handling" is a bit woolly. Maybe: "A stream representing the constants.js file" or similar?

Reply:
Done!